app/src/main/java/com/framgia/englishconversation/utils/binding/BindingUtils.java

```diff
@@ -17,9 +18,12 @@
 import android.text.format.DateUtils;
 import android.view.View;
 import android.widget.ArrayAdapter;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
```

Comment:
ERROR: Unused import: android.widget.CompoundButton. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports